src/component/Button.jsx

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react';
+import PropTypes from 'prop-types';
+
+export default class Button extends Component {
+  constructor(props) {
+    super(props);
+    this.name = props.name;
+    this.value = props.value;
+  }
+
+  render() {
+    return (
+      <button type="button" name={this.name}>{this.value}</button>
```

Comment:
**Great job so far.**
I'm curious why are we having `this.value`, we want to accept a button name as an incoming prop right;
Example:
`<button type="button" >{this.name}</button>`

**_Screenshot from Pathwright below;_**
![image](https://user-images.githubusercontent.com/31897434/82108278-5c849b80-9725-11ea-9337-d58b11b9f9ae.png)